app/controllers/api/v1/users/loans_controller.rb

```diff
@@ -11,4 +11,23 @@ def create
       render json: { error: "Unable to create loan" }, status: 422
     end
   end
+
+  def update
+    user = User.find(params[:id])
+    loans = Loan.where(owner_id: user.id).or(Loan.where(borrower_id: user.id))
+    loan_id = loans.find(params[:loan_id])
```

Comment:
This variable name should change from `loan_id` to `loan` since it is a loan object and not just an id number